### DISCUSSION TITLE
What are the best ways to upload PCAP files from windows 10 my SOC to analyze in my Kibana?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#10507] [Unsupported Versions] [answered]
Hi , i should of said " from windows 10 INTO my SOC"
  I saw that it IS possible to get wireshark PCAP files into my SOC and look at it in kibana. 
https://github.com/Security-Onion-Solutions/securityonion/discussions/10482#discussioncomment-6064055 
But I am confused about this suggestion :
**_You could copy the pcap to your sensor using SCP and run sudo so-import-pcap /full/path/to/import.pcap as described here: https://docs.securityonion.net/en/2.3/so-import-pcap.html_**
 I have googled to find an answer of how to specifically upload PCAP files from windows 10 to my SOC and view them in Kibana?

 thanks for any suggestions or advice

**Comment by dougburks** [comment, marked as answer]
We are currently building a new feature in Security Onion to make this easier. So in the future, you'll be able to upload and import pcap files directly via the SOC web interface. 

In the meantime, there are many ways to secure copy (SCP) a file from Windows to Linux. One option is to use WinSCP:
https://winscp.net/

(If you have questions about WinSCP, please use the WinSCP forum at https://winscp.net/forum/index.php.)

Once you've scp'd the file from Windows to Linux, you can then use so-import-pcap as described at:
https://docs.securityonion.net/en/2.3/so-import-pcap.html

**Reply by iqworks** [reply]
thanks Doug, WinSCP did it for me. now on to sudo so-import-pcap
